### PR TITLE
Tic tac toe game for review

### PR DESCRIPTION
**Pull request by gonjavi**
Things done in this pull request :
- Created the main index.html file.
- Created folders for CSS, and js where the main.js file is.
- Create the folder modules where the factory functions and modules are.
- Added stickler and slintrc.json.

### CODE REVIEW

**Comment by gonjavi** [comment]
4. Stickler does not work and nothing helps  - run eslint in your local env and correct all errors. Remember to let your Code Reviewer know that you had problems with Stickler and you used linter in local env.

According to microverse lintersconfig point 4: https://github.com/microverseinc/linters-config/tree/master/javascript

How it says above stickler does not recognize the repo, I have tried all the ways, and nothing heps. I have run linter on local env.

File: modules/game.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        board_o = 0;
+      });
+      return win; 
+    },
+
+    checkDrawMixin: function(board){
+      let draw = false; 
+      let x = 0;
+      board.forEach(function (square){
+       if (square === "X" || square === "O"){
```
Comment (mcrd25) [inline modules/game.js]: indent space off

File: modules/game.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      });
+      return win; 
+    },
+
+    checkDrawMixin: function(board){
+      let draw = false; 
+      let x = 0;
+      board.forEach(function (square){
+       if (square === "X" || square === "O"){ 
+         x += 1;
```
Comment (mcrd25) [inline modules/game.js]: indent spacing off here

File: modules/game.js (hunk 1 above)
Comment (mcrd25) [inline modules/game.js]: indenting space off here

File: modules/game.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        } 
+        if (x === 9){ 
+          return draw = true;
+        }   
+     });
```
Comment (mcrd25) [inline modules/game.js]: Indent spacing off here

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required 🔍  👀 

Hi @gonjavi, 👋 

Good job with the game's functionality so far! 👍 
I understand your stickler situation and this is fine!
There are just a couple of things you need to address.

### Required Changes
- There should be a game, player and gameboard objects, however, you are missing the gameboard object. See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/objects_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript)
- according to this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/render_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript) YOu are to have render method that rendors the contents of the board array to the webpage.
- Some of your ident spacing is off. Please check it in your code editor. ESlint would help! For example in game.js. Hint: Check your index.html as well
- When I reset the game with `start game` button, the game behaves in a wonky way where it gives the wrong result or I can't click in certain areas of the board (rather it does not update with avatar). Please address this issue. 🙏 

### Additional Suggestions [OPTIONAL]
*The game itself is a bit wonky, maybe there are some improvements that can be made to make it better! I know you are doing this one on your own but don't forget you've got TSEs that can help. 😄*
- [OPTIONAL] I'm currently allowed to play without specifying player name so when the player plays there is clarity on who is X and why. Maybe they should be a default name

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

File: modules/game.js (hunk 2 above)
Comment (gonjavi) [reply modules/game.js]: solved

File: modules/game.js (hunk 1 above)
Comment (gonjavi) [reply modules/game.js]: solved

File: modules/game.js (hunk 3 above)
Comment (gonjavi) [reply modules/game.js]: Solved

**Comment by gonjavi** [comment]
Thank you Maya,

 I have completed the 5 changes required.

File: js/main.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import { Player } from '/modules/player.js';
```
Comment (bellom) [inline js/main.js]: We are missing some `dots` here.
`Error: modules not found.`
image

kindly message on slack if after 5-10 mins you couldn't get to fix it. 👍

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @gonjavi   :clap: 

#### Great work making all the necessary corrections from the previous review `BUT we are having some bugs`.

**Kindly check my comment below.**

Tips: **[Optional]**
Kindly watch the video tutorial on how to create a live link  👉  [Link](https://www.loom.com/share/76088842a2d14322bbcf65b884f4d42d);

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: js/main.js (hunk 4 above)
Comment (gonjavi) [reply js/main.js]: Hi Oluwaseun,

I think I have solved it.

Regards

**Review by bellom — APPROVED** [review]
## Hi @gonjavi  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**